process.md

```diff
@@ -111,6 +111,10 @@ In general, actions taken on proposals should be limited to one per week.  Pleas
 - If the Steering Committee votes to return a proposal for revisions, bringing the revised proposal back into review cannot happen until the following week.  This allows for adequate time to ensure the Steering Committee's agreed upon revisions have been made.  
 - A proposal cannot be brought into review and voted upon in the same week, as Steering Committee representatives would not have sufficient time to review the proposal prior to voting.
 
+### Priority
+
+For many different reasons it can be possible that proposals remain in review for a couple of iterations. While it is important to improve the proposals to contain a complete and mature solution before acceptance these proposals occupy slots for other proposals. Therefore older proposals should be treated with a higher priority. The SDLC members, the proposal author and the PM should commit to support proposals of a higher priority and provide feedback in time to help to complete these proposals. The idle time of these proposals should be avoided as much as possible. Everyone should help to complete proposal and allow new proposals to get into review.
```

Comment:
```suggestion
For many different reasons it can be possible that a proposal remain in review for a couple of iterations. While it is important to improve the proposal to contain a complete and mature solution before acceptance, each proposal occupies a potential slot for other proposals awaiting review. Therefore older proposals should be treated with a higher priority. The SDLC members, the proposal author, and the PM should commit to supporting proposals of a higher priority and providing feedback in time to help to reach a decision on these proposals. The idle time of these proposals should be avoided as much as possible. Everyone should help to reach a decision on a proposal and allow new proposals to enter review.
```